### ISSUE TITLE
Astrometry_net - images failing to solve

### ISSUE THREAD

**Issue by awesomecosmos** [astropy/astroquery#2133]
I am writing code using **astroquery.astrometry_net** package, and am attempting to use **ast.solve_from_image** from a test list of 4 FITS images, 2 with exposure length 60s, and 2 with exposure length 300s. I have found that all but one image fails to solve, and they each return the following error:

```
Traceback (most recent call last):

  File "C:\Users\ave41\OneDrive - University of Canterbury\Master's 2021\ASTR480 Research\ASTR480 Code\01 Data Reduction Pipeline\DataReductionPipeline\src\asp.py", line 93, in <module>
    test_wcs_header = ast.solve_from_image(another_test_img3,solve_timeout=1000,

  File "C:\Users\ave41\Anaconda3\lib\site-packages\astroquery\astrometry_net\core.py", line 406, in solve_from_image
    return self.solve_from_source_list(sources['xcentroid'],

  File "C:\Users\ave41\Anaconda3\lib\site-packages\astroquery\astrometry_net\core.py", line 305, in solve_from_source_list
    return self.monitor_submission(submission_id,

  File "C:\Users\ave41\Anaconda3\lib\site-packages\astroquery\astrometry_net\core.py", line 255, in monitor_submission
    raise TimeoutError('Solve timed out without success or failure',

TimeoutError: ('Solve timed out without success or failure', 4857551)
```

For the image that does work, it is the first file in the directory, and it returns a WCS header as expected, and when I check on the astrometry.net page, it has solved successfully (http://nova.astrometry.net/status/4857455). However, the other three images return the error above, and when I check on astrometry.net at the time when the above error is returned, it says the job is queued, and after 5-10 minutes, fails to solve on the website too. Example of a failed solve: http://nova.astrometry.net/status/4857501

I have set a timeout of 1000 seconds in my code (https://github.com/awesomecosmos/Data-Reduction-Pipeline/blob/main/src/asp.py), and yet the images fail to solve within 3 minutes of running the program. I cannot find any more documentation on how to solve this error. Please can someone point me in the right direction?

**Comment by keflavich** [comment]
Unfortunately, I'm pretty sure this is a limitation of the astrometry.net server & service.  astrometry.net will not solve every field.  This question is more appropriate for the astrometry.net team, as there is no astroquery issue here - astroquery is just reporting what astrometry.net has done.

**Comment by awesomecosmos** [comment]
Hi there, I think I have found where the error is. In line 406 of ```core.py``` (https://github.com/astropy/astroquery/blob/main/astroquery/astrometry_net/core.py) in the function ```solve_from_image```, the return statement:
```
            return self.solve_from_source_list(sources['xcentroid'],
                                               sources['ycentroid'],
                                               ccd.header['naxis1'],
                                               ccd.header['naxis2'],
                                               **settings)
```
should also be passing the ```solve_timeout time``` parameter in the return statement. Since it isn't, it defaults to the value of TIMEOUT. I suspect this issue will be fixed if ```solve_timeout=solve_timeout``` is also passed as a parameter in the return statement.

**Comment by keflavich** [comment]
@awesomecosmos Looks like you're right!  Would you care to submit this proposed change as a pull request?

**Comment by mwcraig** [comment]
I believe this timeout issue was fixed in #2484
